src/test/java/org/broadinstitute/dsde/consent/ontology/integration/Term.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.dsde.consent.ontology.integration;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Term {
```

Comment:
This is so close to `TermResource`, and pretty close to `OntologyTerm` ... any way to consolidate? Maybe not, and it isn't a big deal.